docs/sourcemaps.rst

```diff
@@ -3,69 +3,109 @@
-browsers.
+UglifyJS is a popular tool for minifying your source code for production. It can dramatically
+reduce the size of your files by eliminating whitespace, rewriting variable names, removing dead code branches,
+and more.
 
-One thing to note is that Sentry will attempt to process the source map
-before storing (or grouping) an event. This ensures that if we are able to
-deminify the source, we'll be able to more effectively group similar
-events over time.
+If you are using UglifyJS to minify your source code, the following command will additionally generate a source map
+that maps the minified code back to the original source:
 
-Browser Support
----------------
+::
 
-In our experiences, the only browser that routinely delivers usable error
-reports is **Google Chrome**.
+    node_modules/uglify-js/bin/uglifyjs {input} \
+      --source-map-root={relroot}/ \
+      --source-map-url={name}.map.js \
+      --source-map={relpath}/{name}.map.js -o {output}
 
-For additional information, see this more up-to-date `wiki page
-<https://github.com/ryanseddon/source-map/wiki/Source-maps:-languages,-tools-and-other-info>`_.
 
-Generating a Source Map
------------------------
+Webpack
+~~~~~~~
 
-While there are several compression libraries which support source maps,
-as of writing our recommendation is to use `UglifyJS
-<https://github.com/mishoo/UglifyJS2>`_. That said, many tools such as
-`webpack <http://webpack.github.io/>`_ and `browserify
-<http://browserify.org/>`_.
+Webpack is a powerful build tool that resolves and bundles your JavaScript modules into files fit for running in the
+browser. It also supports many different "loaders" which can convert higher-level languages like TypeScript and
+ES6/ES2015 into browser-compatible JavaScript.
 
-As an example, we can look at how we used to do things with Sentry (pre-webpack):
+Webpack can be configured to output source maps by editing webpack.config.js.
 
 ::
 
-    node_modules/uglify-js/bin/uglifyjs {input} \
-      --source-map-root={relroot}/ \
-      --source-map-url={name}.map.js \
-      --source-map={relpath}/{name}.map.js -o {output}
+    module.exports = {
+        // ... other config above ...
+        entry: {
+          "app": "src/app.js"
+        },
+        output: {
+          path: path.join(__dirname, 'dist'),
+          filename: "[name].js",
+          sourceMapFilename: "[name].map.js",
+        }
+    };
 
-We won't attempt to go into the complexities of source maps, so we
-recommend taking a stab at compiling them, and running them against a
-validator.
+Making Source Maps Available to Sentry
+--------------------------------------
 
-Validating a Source Map
------------------------
+Source maps can be either:
 
-We maintain an online validation tool that can be used to test your source
-(and sourcemaps) against: `sourcemaps.io <http://sourcemaps.io>`_.
+1) Served publicly over HTTP alongside your source files.
```

Comment:
I'd rather put a "recommended" tag by #2. I think #1 logically is the first thing to explain to someone, and it's far easier to set up.